test/sim/data.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const assert = require('./../assert');
+const toID = require('../../.sim-dist/dex').toID;
```

Comment:
I wonder if we should just define `toID` as a global in tests, instead of needing to import it everywhere?

Reply:
Tests already have plenty of globals, anyway, afaik.

Reply:
Yeah, if we already define `Dex` as a global, also defining `toID` isn't _worse_.